relay-cabi/src/auth.rs

```diff
@@ -181,10 +182,13 @@ ffi_fn! {
             max_age,
         )?;
 
+        let version = response.version();
+
         let relay_response = RelayRegisterResponse {
             relay_id: response.relay_id(),
             token: response.token(),
             public_key: state.public_key(),
+            version: version.as_str(),
```

Comment:
You'll also have to add this to the return value of [`validate_register_response`](https://github.com/getsentry/relay/blob/8738ac45f8971909cc9021143a79b85c449b11a9/py/sentry_relay/auth.py#L119-L123).

Reply:
hmm, already there, don't know why we didn't see it on code review.